app/controllers/katello/concerns/api/v2/content_overrides_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Katello
+  module Concerns
+    module Api::V2::ContentOverridesController
+      extend ActiveSupport::Concern
+
+      # overriding_object => pass it either an activation key or content host
+      def validate_content_overrides_enabled(content_params, overriden_object)
+        valid_values = ["yes", "no", "0", "1", "default"]
```

Comment:
I would argue the model only cares about ["1", "0" or nil] . its just a place holder. The yes/no/default these are all mainly a controller formatting thing.